vitest.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+    test: {
+        dir: './test',
+        setupFiles: ['./test/vitest.setup-file.ts'],
+        coverage: { enabled: true },
+        testTimeout: 300_000 // the performance of generating safe primes in JS (without node) are terrible
```

Comment:
you can pass the timeout time for a specific test as a third argument of `test` 
see also, https://vitest.dev/guide/migration.html#timeout

Reply:
it's overall fine to have the global timeout. it does not seem to be applied to all tests though